.github/workflows/build-binaries.yml

```diff
@@ -32,7 +32,7 @@ jobs:
           submodules: recursive
       - uses: actions/setup-python@v5
         with:
-          python-version: "3.12"
+          python-version: "3.13"
```

Comment:
So just to confirm, the `-cp38-` part of the wheel name in PyPi means it is compatible with 3.8+, but in fact the wheel was built using 3.12 (and will now be built with 3.13 and will contain a `-cp39-` name fragment).

Reply:
We tell cibuildwheel to build for these minimums: `cp39-win_amd64`, `cp39-manylinux_x86_64`, `cp39-manylinux_aarch64`, `cp39-macosx_x86_64`, and `cp39-macosx_arm64`. And we tell PyO3 to compile with ABI compat to `abi3-py39`. It is those two tools that make sure during the build-binary step to compile for 3.9 minimum. cibuildwheel sometimes runs inside a container, and it's a certain container they build within to get older libc compat and such (ref https://github.com/pypa/manylinux for Linux for instance). May have to reference those projects for more details or look at the logs of the builds at https://github.com/temporalio/sdk-python/actions/workflows/build-binaries.yml.